### PR TITLE
Updated for June

### PR DESCRIPTION
**Pull request by anchawl**
@levinec

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender/microsoft-secure-score-whats-new.md [hunk 1]
```diff
@@ -30,7 +30,13 @@ ms.technology: m365d
 To make Microsoft Secure Score a better representative of your security posture, we have made some changes. To learn about planned changes, see [What's coming in Microsoft Secure Score?](microsoft-secure-score-whats-coming.md)
 
 Microsoft Secure Score can be found at https://security.microsoft.com/securescore in the [Microsoft 365 security center](overview-security-center.md).
-    
+
+## June 2021
+
+### Remove improvement action related to Microsoft Cloud App Security
+
+- Use Cloud App Security to detect anomalous behavior
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/microsoft-secure-score-whats-new.md]:
```suggestion
- Use Cloud App Security to detect anomalous behavior.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @anchawl, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**